xbmc/network/httprequesthandler/python/HTTPPythonWsgiInvoker.cpp

```diff
@@ -75,26 +76,22 @@ PyObject* PyInit_Module_xbmcwsgi(void);
 
 using namespace PythonBindings;
 
-typedef struct
+namespace
 {
-  const char *name;
-  CPythonInvoker::PythonModuleInitialization initialization;
-} PythonModule;
-
-static PythonModule PythonModules[] =
+// clang-format off
+const std::array<PythonModule, 3> PythonModules =
 {
-  { "xbmc",           PyInit_Module_xbmc },
-  { "xbmcaddon",      PyInit_Module_xbmcaddon },
-  { "xbmcwsgi",       PyInit_Module_xbmcwsgi }
+  PythonModule{ "xbmc",           PyInit_Module_xbmc },
```

Comment:
explicit type shouldn't be needed
```suggestion
 { "xbmc",           PyInit_Module_xbmc },
```

Reply:
The last patch by @neo1973 simplifies this issue.